packages/varlet-ui/src/card/docs/zh-CN.md

```diff
@@ -180,6 +180,7 @@ const floating = ref(false)
 | 插槽名           | 说明      | 参数 |
 |---------------|---------| ---- |
 | `image`       | 自定义图片   | `-`  |
+| `image`       | 自定义卡片内容   | `-`  |
```

Comment:
The slot name is incorrect and please bump a version.

Reply:
done